readme.md

```diff
@@ -757,6 +757,7 @@
 - [Algorand](https://github.com/aorumbayev/awesome-algorand#readme) - An open-source, proof of stake blockchain and smart contract computing platform.
 - [ZeroNet](https://github.com/zolagonano/awesome-zeronet#readme) - A decentralized web-like network of peer-to-peer users.
 - [Cosmos SDK](https://github.com/cosmos/awesome-cosmos#readme) - Modular framework for building app-specific blockchains in Go.
+- [Tor](https://github.com/polycarbohydrate/awesome-tor#readme) - Tor is a free overlay network for enabling anonymous communication.
```

Comment:
The description should not start with the list name.

Reply:
Fixed.

Reply:
ok

Reply:
sosking97

Reply:
> The description should not start with the list name.